### ISSUE TITLE
Manually created jobs via the `Trigger CronJob` function are missing parent labels

### ISSUE THREAD

**Issue by goenning** [aptakube/aptakube#296]
### Current and expected behavior

kubectl inserts these labels when triggering a new cronjob, but Aptakube does not.

We should make our implementation match what kubectl is doing.

```
...
  labels:
    batch.kubernetes.io/controller-uid: 93713796-90fc-43a2-adba-0e647a458081
    batch.kubernetes.io/job-name: test
    controller-uid: 93713796-90fc-43a2-adba-0e647a458081
    job-name: test
```

### Environment & Operating System

N/A

**Comment by krmichelos** [comment]
The more important thing is that the ownerReference is missing the `controller: true` field/value

**Comment by goenning** [comment]
Fixed on 1.7.7, thanks for sharing the details @krmichelos
